scripts/deploy.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+[[ "$TRACE" ]] && set -x
+
+start_viewer_sinatra() {
+  mkdir -p /tmp/viewer-sinatra
+  cd /tmp/viewer-sinatra
+  # TODO: Make `master` configurable
+  curl -fsSL https://github.com/everypolitician/viewer-sinatra/archive/master.tar.gz | tar -x -f - --strip 1
+  bundle install
+  bundle exec ruby app.rb &
+  while ! nc -z localhost 4567; do sleep 1; done
+}
+
+build_viewer_static() {
+  cd /tmp
+  wget -nv -m localhost:4567/status/all_countries.html || (echo "wget exited with non-zero exit code: $?" >&2 && exit 1)
+}
+
+deploy_viewer_static() {
+  cd /tmp
+  git clone https://github.com/everypolitician/viewer-static.git
```

Comment:
Ah yes, that's a very good idea. Amazingly, according to git, a shallow clone with `--depth 1` _still_ results in a 13.68 MiB clone, but I suppose that's much better than 700+MB! Will add that now.